nano/templates/supermatter_monitor.tmpl

```diff
@@ -56,7 +56,33 @@
 				{{else}}
 					<span class='good'>{{:data.SM_EPR}}</span>
 				{{/if}}
-			</div>		
+			</div>
+			<div class="itemLabel">
+				Crystal Ph Suppression
+			</div>
+			<div class="itemContent">
+				{{if data.SM_PHO = 100}}
+					<span class='bad'>{{:data.SM_PHO}}</span>
+				{{else data.SM_PHO > 50}}
+					<span class='average'>{{:data.SM_PHO}}</span>
+				{{else data.SM_PHO < 30}}
+					<span class='average'>{{:data.SM_EPR}}</span>
```

Comment:
```suggestion
					<span class='average'>{{:data.SM_PHO}}</span>
```
You'll want this one too if you're doing it through github exclusively.

Reply:
Thanks.